com.unity.perception/Runtime/GroundTruth/PerceptionCamera_InstanceSegmentation.cs

```diff
@@ -29,12 +29,26 @@ partial class PerceptionCamera
         RenderTexture m_InstanceSegmentationTexture;
         RenderTextureReader<uint> m_InstanceSegmentationReader;
 
+    #if HDRP_PRESENT
+        InstanceSegmentationPass m_InstanceSegmentationPass;
+        LensDistortionPass m_LensDistortionPass;
+    #elif URP_PRESENT
+        InstanceSegmentationUrpPass m_InstanceSegmentationPass;
+        LensDistortionUrpPass m_LensDistortionPass;
+    #endif
+
+        public void OverrideLensDistortionIntensity(float intensity)
```

Comment:
If this is only for testing, could the tests use Volume and set the lens distortion directly? Or, could this be made internal?